piptools/utils.py

```diff
@@ -380,3 +380,13 @@ def create_install_command():
     from pip._internal.commands import create_command
 
     return create_command("install")
+
+
+def get_trusted_hosts(finder):
+    """
+    Returns a list of trusted hosts from given finder.
+    """
+    if PIP_VERSION < (19, 2):
+        return [host for _, host, _ in finder.secure_origins]
+
+    return finder.trusted_hosts
```

Comment:
Alternatively
```suggestion
    return list(finder.trusted_hosts)
```
so that the return type is consistent.

Reply:
I'd prefer a generator.

Reply:
Addressed in 7a54db3